### PR TITLE
fix(odd): medium odd button design qa corrections

### PR DESCRIPTION
**Pull request by ewagoner**
# Overview

This PR addresses design qa notes for RAUT-361, ODD Medium Buttons

# Test Plan

Changes are mostly in storybook, and are otherwise covered by unit tests.

# Changelog

- Renamed button types
- Added more options to storybook stories
- Removed unnecessary storybook examples

# Review requests

None.

# Risk assessment

None.

### CODE REVIEW

File: app/src/atoms/buttons/OnDeviceDisplay/__tests__/MediumButton.test.tsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ describe('MediumButton', () => {
   it('renders the tertiary light button', () => {
```
Comment (koji) [inline app/src/atoms/buttons/OnDeviceDisplay/__tests__/MediumButton.test.tsx]: Could you update the description?
```suggestion
  it('renders the tertiary low light button', () => {
```

Comment (ewagoner) [reply app/src/atoms/buttons/OnDeviceDisplay/__tests__/MediumButton.test.tsx]: Done!

File: app/src/atoms/buttons/OnDeviceDisplay/__tests__/MediumButton.test.tsx [hunk 2]
```diff
@@ -54,18 +54,18 @@ describe('MediumButton', () => {
       `background-color: ${COLORS.red_three}`
     )
   })
-  it('renders the tertiary button', () => {
+  it('renders the tertiary high button', () => {
```
Comment (koji) [inline app/src/atoms/buttons/OnDeviceDisplay/__tests__/MediumButton.test.tsx:57]: 👍

**Review by koji — APPROVED** [review]
lgtm!

**Pull request merged**